mep/people/views.py

```diff
@@ -244,6 +249,29 @@ def get_context_data(self, **kwargs):
                 'endDate': end.isoformat()
             } for start, end in account.event_date_ranges]
         }
+
+        # plottable locations for member address map visualization
+        # addresses can be stored on either Person or Account
+        addresses = Address.objects.filter(Q(account__pk=account.pk) |
+            Q(person__pk=self.object.pk)) \
+            .filter(location__latitude__isnull=False) \
```

Comment:
Should this be a queryset filter on `Address` ?

Reply:
if we can think of anyplace else it might get used, maybe? but i couldn't

Reply:
On the member list map

Reply:
the one for the big map is probably gonna be on `Person`, right?

Reply:
Hopefully it will be accessed from Person, but will still need to use the address queryset somewhere, whether at indexing time or in a json view serving out addresses for the member map.  Like I said, might not matter for this version but we should think about refactoring later.